tests/support/loadJsdom.ts

```diff
@@ -31,6 +31,13 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 
 global.cancelAnimationFrame = () => {};
 
+global.IntersectionObserver = function () {
```

Comment:
Only thought is that we would want `jsdom` as a dev dependency, but then we have a dev only module of `loadJsdom`?  That would essentially make it a hard dependency, right?  Even though it would be built out, it feels a little bit _dirty_, IMO.